### ISSUE TITLE
UI issues in Linux Mint with HiDPI

### ISSUE THREAD

**Issue by linuxmintquestion** [StevensNJD4/LazyMan#46]
Hello, I am experiencing UI issues running LazyMan under the latest version of Linux Mint with a HiDPI display. I followed the LazyMan installation instructions exactly. I've repeated the installation steps on a laptop also running the latest version of Linux Mint and did not have the same issue.

The UI is unusable on the HiDPI display in that I cannot select the "PLAY" button. I can use the drop-down menus to change "FEED", "QUALITY", and "CDN", but they are noticeable cut off on the bottom of the screen. I can even see the "Save Stream" checkbox. Adjusting the window, maximizing, or even changing the Desktop's resolution does not appear to have any effect, I still cannot see the "PLAY" button to select a stream.

Any ideas about what I might be able to do to remedy this? Even if there were a simple shortcut key to play a stream that would work. I can select game matchups, if only I could press "P" or "Enter" with my desired matchup selected that would work to launch the stream in VLC...

Help?

**Comment by linuxmintquestion** [comment]
@StevensNJD4 Great app btw - I used it all last season on this same PC and same HiDPI display. The only change is I was previously on an older version of Linux Mint and this was a fresh install of 19.3. I also have weird UI scaling issues with VLC now under 19.3, but VLC is still usable, it just has crazy tiny menus...

**Comment by StevensNJD4** [comment]
not an ideal solution but does resizing the window help?

as for vlc, i don't develop that nor run linux

**Comment by linuxmintquestion** [comment]
Nope. No matter how I resize it or maximize it the very bottom of the screen appears cut off and I cannot see nor select the "PLAY" button.

I'm not much of a coder, is there a way for me to edit some config files to add a shortcut key to launch the highlighted matchup?

(The VLC comment was just in case VLC/LazyMan share any common gui libraries... I *think* my VLC issue is related to qt5...)

**Comment by Pacane** [comment]
is VLC not displaying properly or is it the LazyMan app?

**Comment by linuxmintquestion** [comment]
@Pacane Both. Unrelated, VLC has given me issues since I first booted up Linux Mint 19.3. I'm dealing with it though, I navigate using shortcuts anyway (though the miniature Seek Bar _is_ annoying).

But LazyMan itself has a tiny UI (and the spacing is all jacked up too). I'll screenshot it tonight.

**Comment by StevensNJD4** [comment]
@linuxmintquestion you could double click the game or hit alt+p

**Comment by Pacane** [comment]
@linuxmintquestion what desktop environment are you using? Also are you using Xorg or Wayland? There are environment variables you can try to override to force some apps to use hi dpi scaling.

**Comment by linuxmintquestion** [comment]
@StevensNJD4 **Alt+P** is probably the solution I'm looking for!! I didn't know the shortcut... (Double-Clicking the game didn't launch it though, I'll try it out tonight)

@Pacane I _think_ Linux Mint moved to Wayland with version 19. Which is when my HiDPI issues began, so I bet that is the source of my problems... I'll do some Googling later, if you have any resources though I'd appreciate it

Thanks everyone :-)

**Comment by Pacane** [comment]
@linuxmintquestion The best article about hidpi is probably [this one](https://wiki.archlinux.org/index.php/HiDPI)

**Comment by linuxmintquestion** [comment]
Thank you. I'll see what I can come up with. I was wrong about Wayland, it is x11.

**Comment by linuxmintquestion** [comment]
> @linuxmintquestion you could double click the game or hit alt+p

So neither **Alt+P** nor **Double-Click** launch VLC with a game selected
